openlibrary/catalog/marc/tests/test_data/xml_input/39002054008678.yale.edu_marc.xml

```diff
@@ -1 +1 @@
/slim" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.loc.gov/MARC21/slim &#xA;http://www.loc.gov/standards/marcxml/schema/MARC21slim.xsd" type="Bibliographic"><marc:leader>00733cam a2200265 a 4500</marc:leader><marc:controlfield tag="001">2072764</marc:controlfield><marc:controlfield tag="005">20020514093448.0</marc:controlfield><marc:controlfield tag="008">830301s1898    oncabc        000 0 eng  </marc:controlfield><marc:datafield tag="010" ind1=" " ind2=" "><marc:subfield code="a">   02012591  </marc:subfield></marc:datafield><marc:datafield tag="035" ind1=" " ind2=" "><marc:subfield code="a">(OCoLC)ocm09268563</marc:subfield></marc:datafield><marc:datafield tag="035" ind1=" " ind2=" "><marc:subfield code="9">DEC3523YL</marc:subfield></marc:datafield><marc:datafield tag="040" ind1=" " ind2=" "><marc:subfield code="a">DLC</marc:subfield><marc:subfield code="c">BOS</marc:subfield><marc:subfield code="d">MHS</marc:subfield><marc:subfield code="d">YUS</marc:subfield></marc:datafield><marc:datafield tag="043" ind1=" " ind2=" "><marc:subfield code="a">n-cn-on</marc:subfield></marc:datafield><marc:datafield tag="049" ind1=" " ind2=" "><marc:subfield code="a">YUSW</marc:subfield></marc:datafield><marc:datafield tag="050" ind1="0" ind2=" "><marc:subfield code="a">F1058</marc:subfield><marc:subfield code="b">.C74</marc:subfield></marc:datafield><marc:datafield tag="099" ind1=" " ind2=" "><marc:subfield code="a">Ca17</marc:subfield><marc:subfield code="a">40</marc:subfield></marc:datafield><marc:datafield tag="100" ind1="1" ind2=" "><marc:subfield code="a">Conant, Thomas,</marc:subfield><marc:subfield code="d">1842-1905.</marc:subfield></marc:datafield><marc:datafield tag="245" ind1="1" ind2="0"><marc:subfield code="a">Upper Canada sketches /</marc:subfield><marc:subfield code="c">by Thomas Conant.</marc:subfield></marc:datafield><marc:datafield tag="260" ind1=" " ind2=" "><marc:subfield code="a">Toronto :</marc:subfield><marc:subfield code="b">W. Briggs,</marc:subfield><marc:subfield code="c">1898.</marc:subfield></marc:datafield><marc:datafield tag="300" ind1=" " ind2=" "><marc:subfield code="a">243 p., [27] leaves of plates :</marc:subfield><marc:subfield code="b">ill., map, port. ;</marc:subfield><marc:subfield code="c">24 cm.</marc:subfield></marc:datafield><marc:datafield tag="651" ind1=" " ind2="0"><marc:subfield code="a">Ontario</marc:subfield><marc:subfield code="x">History.</marc:subfield></marc:datafield><marc:datafield tag="651" ind1=" " ind2="0"><marc:subfield code="a">Ontario</marc:subfield><marc:subfield code="x">Description and travel.</marc:subfield></marc:datafield><marc:datafield tag="927" ind1=" " ind2=" "><marc:subfield code="a">20000807R </marc:subfield></marc:datafield><marc:datafield tag="928" ind1=" " ind2=" "><marc:subfield code="a">ACOC0600</marc:subfield></marc:datafield><marc:datafield tag="948" ind1=" " ind2=" "><marc:subfield code="a">RECON</marc:subfield></marc:datafield></marc:record>
```

Comment:
What was changed in this test file?  The diff does not highlight any change.

If you add one and only one carriage return (\n) to the end of the file the GitHub will remove the ⛔.